pkg/kubelet/kubelet_node_status.go

```diff
@@ -572,6 +571,19 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 				}
 			}
 		}
+
+		if utilfeature.DefaultFeatureGate.Enabled(features.DevicePlugins) {
```

Comment:
The mock structure would also remove this if

Reply:
@jiayingz Local storage does not have a stub concept. We do not need this check here as @RenaudWasTaken pointed out.

Reply:
removed now.